src/ui/flamegraph.rs

```diff
@@ -82,6 +82,7 @@ mod tests {
     }
 }
 
+#[cfg(unix)]
```

Comment:
can you add a comment saying why this is cfg'ed out? also maybe `#[cfg(not(windows))]` depending? Is this because of the lack of perl?

Reply:
It's a combination of lack of perl, and that the tempdir.close() function fails with a 'directory is not empty' error on windows. I've updated the comment to track this.